docs/changelog.md

```diff
@@ -7,6 +7,7 @@ All notable changes to this project will be documented in this file.
 * Unit test code coverage improved to ~90%
 * Notifications
   * Added AWS_SNS delivery method type
+* Improved CDK code (for tooling account part) readability and manageability by switching to Nested Stacks approach.
```

Comment:
should we mention in our docs about the upgrade process (from ver1.0 to 1.1)? or is it just about running the cdk deploy command?

Reply:
Usual cycle of cdk destroy / cdk deploy should be sufficient here, so I guess we can leave the update statement "as is".